src/main/resources/coremods/method_redirector.js

```diff
@@ -0,0 +1,77 @@
+var ASMAPI = Java.type('net.minecraftforge.coremod.api.ASMAPI');
+var Opcodes = Java.type('org.objectweb.asm.Opcodes');
+var Handle = Java.type('org.objectweb.asm.Handle');
+var MethodInsnNode = Java.type('org.objectweb.asm.tree.MethodInsnNode');
+
+function finalizeSpawnNode(node){
+    return new MethodInsnNode(
+        Opcodes.INVOKESTATIC, 
+        "net/minecraftforge/event/ForgeEventFactory", 
+        "onFinalizeSpawn", 
+        "(Lnet/minecraft/world/entity/Mob;Lnet/minecraft/world/level/ServerLevelAccessor;Lnet/minecraft/world/DifficultyInstance;Lnet/minecraft/world/entity/MobSpawnType;Lnet/minecraft/world/entity/SpawnGroupData;Lnet/minecraft/nbt/CompoundTag;)Lnet/minecraft/world/entity/SpawnGroupData;", 
+        false);
+}
+
+function contains(list, target) {
+	for(var i = 0; i < list.length; i++) {
+		if(list[i] == target) return true;
+	}
+	return false;
```

Comment:
Formatting: 4 spaces for indentation.
```suggestion
    for(var i = 0; i < list.length; i++) {
        if(list[i] == target) return true;
    }
    return false;
```